.github/workflows/ci.yml

```diff
@@ -31,3 +31,26 @@ jobs:
       - run: yarn build
 
       - run: yarn test
+
+  test-apollo-3:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 20.x]
```

Comment:
`@apollo/client@4` uses null chainging (added in node 14) and `||=` operator (added in node 15), so have to drop those versions from testing for it.